src/test/java/net/openhft/chronicle/bytes/perf/ByteArrayReadWriteJLBH.java

```diff
@@ -0,0 +1,193 @@
+ *
+ *     https://chronicle.software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.openhft.chronicle.bytes.perf;
+
+import net.openhft.chronicle.bytes.Bytes;
+import net.openhft.chronicle.core.util.NanoSampler;
+import net.openhft.chronicle.jlbh.JLBH;
+import net.openhft.chronicle.jlbh.JLBHOptions;
+import net.openhft.chronicle.jlbh.JLBHTask;
+import net.openhft.chronicle.jlbh.TeamCityHelper;
+
+import static java.lang.System.setProperty;
+
+public class ByteArrayReadWriteJLBH implements JLBHTask {
+
+    public static final int SHORT_LENGTH = 37;
+    public static final int LONG_LENGTH = 1_024;
+    private static final int ARRAY_SIZE = 1024 * 50;
+    private static final byte[] BYTE_ARRAY = new byte[ARRAY_SIZE];
+    private static final int ITERATIONS = 50_000;
+
+    private final Bytes<?> bytesImpl;
+    private JLBH jlbh;
+    private NanoSampler absoluteRead;
+    private NanoSampler absoluteWrite;
+    private NanoSampler relativeRead;
+    private NanoSampler relativeWrite;
+    private NanoSampler marshalRead;
+    private NanoSampler marshalWrite;
+    private Bytes<?> targetBytes;
+    private NanoSampler copyTo;
+    private int length;
+    private long lengthOfAsciiString;
+    private String stringForLength;
+
+    public ByteArrayReadWriteJLBH(Bytes<?> bytes, int length) {
+        this.bytesImpl = bytes;
+        this.length = length;
+    }
+
+    public static void runForBytes(Bytes<?> bytes) {
+        runForBytes(bytes, LONG_LENGTH);
+    }
+
+    public static void runForBytes(Bytes<?> bytes, int length) {
+        setProperty("jvm.resource.tracing", "false");
+        new JLBH(new JLBHOptions()
+                .warmUpIterations(15_000)
+                .iterations(ITERATIONS)
+                .runs(3)
+                .recordOSJitter(false)
+                .throughput(5_000)
+                .pauseAfterWarmupMS(100)
+                .accountForCoordinatedOmission(true)
+                .jlbhTask(new ByteArrayReadWriteJLBH(bytes, length)))
+                .start();
+    }
+
+    @Override
+    public void init(JLBH jlbh) {
+        this.jlbh = jlbh;
+        bytesImpl.write8bit(stringForLength(length));
+        absoluteRead = jlbh.addProbe("absoluteRead");
+        absoluteWrite = jlbh.addProbe("absoluteWrite");
+        relativeRead = jlbh.addProbe("relativeRead");
+        relativeWrite = jlbh.addProbe("relativeWrite");
+        marshalRead = jlbh.addProbe("marshalRead");
```

Comment:
would be good to have "8bit" in the name, and, as discussed, it would be very useful to have utf8 versions in here for https://github.com/OpenHFT/Chronicle-Bytes/issues/203